docs/source/upcoming_release_notes/1161-MSC2_PV_returning_SmarAct_device_class.rst

```diff
@@ -0,0 +1,30 @@
+1161 MCS2 PV returning SmarAct device class
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- _GetMotorClass: Add MCS2 PV to motor_types and link it to SmarAct device class
```

Comment:
nitpick: I think referring to private methods here is not correct for release notes, we should instead focus on the relevant user context, e.g. the result of this fix: that the `Motor` function will properly discover MCS2 SmarAct devices.

Reply:
I also might consider this a bugfix but it's ambiguous enough that I'm not sure